src/doc/rustdoc/src/documentation-tests.md

```diff
@@ -3,6 +3,9 @@
 `rustdoc` supports executing your documentation examples as tests. This makes sure
 that examples within your documentation are up to date and working.
 
+Before going any further, please note that only documentation examples in the source
+code are run. So if you have ones in your `tests` folder files, they won't be run.
```

Comment:
This seems like it should be part of the Cargo docs or the book. Rustdoc doesn't assign any meaning to src/ or tests/.

Reply:
Or in other words, if you use `rustdoc --test` directly it works fine. It's only going through cargo that breaks.
```
rustdoc --test tests/doc.rs 

running 1 test
test tests/doc.rs - (line 1) ... FAILED

failures:

---- tests/doc.rs - (line 1) stdout ----
Test executable failed (exit code 101).

stderr:
thread 'main' panicked at 'assertion failed: `(left == right)`
  left: `1`,
 right: `2`', tests/doc.rs:3:1
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
That's a good point. I'll comment that on the issue. Closing this PR then!